src/recipe.rst

```diff
@@ -26,7 +26,7 @@ Step-by-step instructions
 1. Fork the `example recipes <https://github.com/conda-forge/staged-recipes/tree/master/recipes>`_
 2. Within your forked copy, generate a new folder in the recipes subdirectory and copy `meta.yml <https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml>`_ from the example directory. Please leave the example directory unchanged!
 3. Edit the copied recipe (meta.yml) as you need it
-4. Generate the ssh 256 key like described in the example recipe using the `openssl` tool. As an alternative you can also go to the package description on `PyPi <https://pypi.org>`_ where you can directly copy the ssh 256 key from.
+4. Generate the sha256 key like described in the example recipe using the `openssl` tool. As an alternative you can also go to the package description on `PyPi <https://pypi.org>`_ where you can directly copy the ssh 256 key from.
```

Comment:
I promise I'm not trying to be a jerk, but there's one more towards the end of the line: ssh 256 -> sha256